README.md

```diff
@@ -79,6 +86,34 @@ you will need to copy the settings described below into both of your
 configurations, as keeping them separate means that they cannot follow each
 other automatically.
 
+#### Without flakes
+
+If you haven't enabled flakes yet or don't want to use this feature,
+you can configure Home Manager to use Stylix by importing the
+`stylix.nix` file below (using Home Manager's way of importing files
+`imports = [./stylix.nix]`)
+
+```nix
+{pkgs, ...}@args:
+
+{
+  stylix = {
+    image = ./wallpaper.jpg;
+  };
+
+  imports = [
+    (import ../../stylix/stylix/hm {
+      palette-generator = pkgs.callPackage ../../stylix/palette-generator { };
+      base16 = (import ../../base16) args;
+    })
+  ];
```

Comment:
Could this be a module within the repository, which users import, so they don't specify `palette-generator` and `base16` by hand? That way we can still add or remove arguments without breaking anyone's configuration.

[flake-compat](https://github.com/edolstra/flake-compat) should be able to fetch `base16.nix` automatically, without needing flakes to be enabled.

Reply:
I think that is a good idea.